### ISSUE TITLE
Audit #1 – Share a single file – Options in "Rolle" dropdown not keyboard accessible. Ablaufdatum Datepicker: not reachable via keyboard

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4790]
Connected to Audit 1, see #4300 
1.5 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
Option in "Rolle" dropdown aren't reachable via keyboard. Ablaufdatum Datepicker not at all reachable via keyboard.

# Remediation
Rolle: Building a custom select close to https://github.com/microsoft/sonder-ui/tree/master/src/components/select OR at least suggestion in #4407
Ablaufdatum Datepicker: At least Modal-like focusmanagement (on opening, sending focus into the picker, on close sending back to trigger). **Datepicker is only consisting of divs, failing multiple test steps! 1.3.1, 2.1.1, 2.4.3, 4.2.1, etc**. Needs work!

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
